### ISSUE TITLE
Add official Synology's source

### ISSUE THREAD

**Issue by szyb** [szyb/synopackage_dotnet#80]
Investigate whether it is possible or not.
1. What is the url to package server?
2. Does request and result are in the same format as third-party servers?

Any help is welcome.

**Comment by publicarray** [comment]
Hi 👋
Just wanted to let you know that the official package repository can be queried like this

```
curl -sv -L 'https://payment.synology.com/api/getPackageList.php?package_update_channel=beta&unique=synology_apollolake_418play&build=24922&language=enu&major=6&micro=2&arch=armada37xx&minor=2&nano=4' |jq
```

Just note that it redirects to

```
curl -s 'https://pkgautoupdate.synology.com/getList/v0/6/2/2/24922/4/armada37xx/beta/synology_apollolake_418play/enu' | jq
```

**Comment by szyb** [comment]
Thank you @publicarray !
This is a good place to start. My first tests shows that there are some issues, but the link is working well!
This would be a huge step forward and gives me motivation to make more progress!

**Comment by szyb** [comment]
The first link is very common for all third party servers so I used it and it just work! The main work was done around this, because this is not a third party server like others, so I had to change a bit some pages to make it clear that users don't need to add this repository to their DSM. So I'm not showing this URL on the web page. The official source is browseable and searchable as others.

This change is online now!  Thanks to you @publicarray !
